packages/data-connect/test/util.ts

```diff
@@ -24,13 +24,13 @@ import {
   getDataConnect
 } from '../src';
 
-export const USE_EMULATOR = true;
 export const EMULATOR_PORT = process.env.DC_EMULATOR_PORT;
-// export const EMULATOR_PROJECT = process.env.PROJECT;
-export const CONNECTOR_NAME = 'c';
-export const LOCATION_NAME = 'l';
-export const SERVICE_NAME = 'l';
-export const PROJECT_ID = 'p';
+// eslint-disable-next-line no-console
```

Comment:
Can this `eslint-disable-next-line no-console` comment be removed? It doesn't appear on the surface to be pertinent to the next line.

Reply:
Removed